cli/src/migrations/pgroll.ts

```diff
@@ -1,9 +1,10 @@
 import { Schemas, XataApiClient } from '@xata.io/client';
+import { migrationsDir, readMigrationsDir } from './files.js';
```

Comment:
We should maybe have a shared util package that can be used in the front end as well? A lot of this is copied.

Reply:
I created this since it should probably be a separate task https://github.com/xataio/client-ts/issues/1438